src/index.js

```diff
@@ -5,14 +5,13 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import './css/style.css';
 import 'bootstrap';
 
-import {render} from './js/hierarchy_vis.js';
+import { render } from './js/network-vis.js';
 import init_modal from './js/new_hierarchy_modal.js';
 
-
 // main
-document.addEventListener("DOMContentLoaded", function(){
+document.addEventListener('DOMContentLoaded', function () {
   init_modal();
-  
+
   //load data
   render(sample_data);
```

Comment:
render for both vis's should happen here. We should not use network-vis as an entrypoint for both visualizations. If we do that, we need to create a master vis component that internally renders both. I can see that, but don't just slap hierarchy vis logic into the network-vis components.

Ultimately, code for the hierarchy_vis and the network-vis should be entirely independent.

Reply:
Agree. Needs a complete restructure of the code; else there will be a performance hit iterating through 10s of thousands of data blocks multiple times. Will do that.